lisa/sut_orchestrator/azure/features.py

```diff
@@ -1841,6 +1844,7 @@ def create_or_update(
         settings: Optional[Dict[str, Any]] = None,
         protected_settings: Any = None,
         suppress_failures: Optional[bool] = None,
+        timeout: int = sys.maxsize,
```

Comment:
Don't put max, it's dangerous to wait forever.

Reply:
It was the existing behavior, I added it as a parameter. Shall I reduce it to 20 or 30 minutes?